### PR TITLE
[CCP-174] Made session timeout as 4 hours

### PR DESCRIPTION
**Pull request by mkarthik90**

### CODE REVIEW

File: Conceptpower+Spring/src/main/webapp/WEB-INF/web.xml [hunk 1]
```diff
@@ -60,5 +60,10 @@
 		<servlet-name>appServlet</servlet-name>
 		<url-pattern>/</url-pattern>
 	</servlet-mapping>
+    
+    <!--  Added session timeout as one hour -->
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/web.xml]: comment needs to be corrected. for next time, the branch should have been called 'hotfix' to distinguish from regular bugs.

**Comment by mkarthik90** [comment]
@jdamerow  I updated the comment. I was not able to update the branch name. From next branch creation I will make sure naming conventions

File: Conceptpower+Spring/src/main/webapp/WEB-INF/web.xml (hunk 1 above)
Comment (mkarthik90) [inline Conceptpower+Spring/src/main/webapp/WEB-INF/web.xml]: @jdamerow  Sorry i forgot to change the comment

**Comment by mkarthik90** [comment]
@jdamerow Addressed review comments

**Pull request merged**